integreat_cms/static/src/js/utils/mfa-utils.ts

```diff
@@ -22,10 +22,14 @@ type CredentialResponseFromServer = {
     };
 };
 
-// Based on https://github.com/duo-labs/py_webauthn/blob/master/flask_demo/static/js/webauthn.js
+// Based on https://github.com/MasterKale/SimpleWebAuthn/blob/master/packages/browser/src/helpers/bufferToBase64URLString.ts
 export const b64enc = (buf: Uint8Array) =>
     base64js.fromByteArray(buf).replace(/\+/g, "-").replace(/\//g, "_").replace(/=/g, "");
 
+// Based on https://github.com/MasterKale/SimpleWebAuthn/blob/master/packages/browser/src/helpers/base64URLStringToBuffer.ts
+const b64dec = (b64url: string) =>
+    Uint8Array.from(atob(b64url.replace(/-/g, "+").replace(/_/g, "/")), (c) => c.charCodeAt(0));
```

Comment:
In my opinion, it would be beneficial to include a small test that not only validates the function but also offers additional documentation.

Reply:
I believe the methods could benefit from having a more verbose naming, such as 'base64Encode.' While this may seem obvious to experienced developers, it could provide assistance to newcomers.

Reply:
I added a test in #2747 :)